src/Network/Network/PrivateLinkService/Models/PrivateEndpointConnection.cs

```diff
@@ -0,0 +1,91 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Commands.Network.PrivateLinkService.Models
+{
+    /// <summary>
+    /// A private endpoint connection
+    /// </summary>
+    [Rest.Serialization.JsonTransformation]
+    public partial class PrivateEndpointConnection : ProxyResource
+    {
+        /// <summary>
+        /// Initializes a new instance of the PrivateEndpointConnection class.
+        /// </summary>
+        public PrivateEndpointConnection()
+        {
+            CustomInit();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the PrivateEndpointConnection class.
+        /// </summary>
+        /// <param name="id">Resource ID.</param>
+        /// <param name="name">Resource name.</param>
+        /// <param name="type">Resource type.</param>
+        /// <param name="privateEndpoint">Private endpoint which the connection
+        /// belongs to.</param>
+        /// <param name="privateLinkServiceConnectionState">Connection state of
+        /// the private endpoint connection.</param>
+        /// <param name="provisioningState">State of the private endpoint
+        /// connection.</param>
+        public PrivateEndpointConnection(string id = default(string), string name = default(string), string type = default(string), PrivateEndpointProperty privateEndpoint = default(PrivateEndpointProperty), PrivateLinkServiceConnectionStateProperty privateLinkServiceConnectionState = default(PrivateLinkServiceConnectionStateProperty), string provisioningState = default(string))
```

Comment:
nit: usually we use null as default value for reference object instead of default(xxx), I guess this is generated code, so should be fine.

Reply:
copied from generated code.